app/models/conversion/project.rb

```diff
@@ -22,6 +22,12 @@ def self.policy_class
   scope :with_academy_urn, -> { where.not(academy_urn: nil) }
   scope :by_conversion_date, -> { ordered_by_significant_date }
 
+  MANDATORY_CONDITIONS = [
```

Comment:
What do you think about this being `MANDATORY_TASKS` to make it closer to what it is doing?

Reply:
I was going to make it MANDATORY_TASKS but the "date in the past" isn't a task, I think? Or maybe it's pretty much a task even if it's a project attribute and I'm just splitting hairs

Reply:
Yeah, okay! the rename wasn't a blocker it was the WIP in the commit! 👍